### PR TITLE
DOC: impedances may be obtained for BrainVision

### PR DESCRIPTION
**Pull request by sappelhoff**
This was not clear from the documentation before.

### CODE REVIEW

**Review by adam2392 — APPROVED** [review]
LGTM. Would it be worthwhile to mention that it will be `None` if nothing was read re impedances?

File: mne/io/brainvision/brainvision.py [hunk 1]
```diff
@@ -63,6 +63,9 @@ class RawBrainVision(BaseRaw):
 
     Notes
     -----
+    If the BrainVision header file contained impedance measurements, these may be
```
Comment (cbrnr) [inline mne/io/brainvision/brainvision.py:66]:
```suggestion
    If the BrainVision header file contains impedance measurements, these may be
```

File: mne/io/brainvision/brainvision.py [hunk 2]
```diff
@@ -977,6 +980,9 @@ def read_raw_brainvision(
...
```
Comment (cbrnr) [inline mne/io/brainvision/brainvision.py:983]:
```suggestion
    If the BrainVision header file contains impedance measurements, these may be
```

**Review by cbrnr — APPROVED** [review]
LGTM. I don't think mentioning that this attribute is `None` if no impedance values are available is necessary.

File: mne/io/brainvision/brainvision.py [hunk 3]
```diff
@@ -63,6 +63,9 @@ class RawBrainVision(BaseRaw):
 
     Notes
     -----
+    If the BrainVision header file contains impedance measurements, these may be
+    accessed using ``raw.impedances`` after reading using this function.
```
Comment (scott-huberty) [inline mne/io/brainvision/brainvision.py:67]: Question: I see that this attribute was discussed and added many years ago in #7974, but I thought that attributes that don't survive I/O roundtrips aren't supposed to be added?

e.g. https://github.com/mne-tools/mne-python/issues/11825#issue-1816752541

File: mne/io/brainvision/brainvision.py [hunk 4]
```diff
@@ -977,6 +980,9 @@ def read_raw_brainvision(
...
```
[the remark this answers is not in the document]
Comment (sappelhoff) [reply mne/io/brainvision/brainvision.py]: adressed in https://github.com/mne-tools/mne-python/pull/12861/commits/0be389b216cf97497fff003edaaaca62f4940597

**Comment by sappelhoff** [comment]
> I think we made an exception for BV at one point. But thinking about it I think I'd rather deprecate this param and show an example of how to use pybv directly to access this information.

I agree in principle @larsoner and I'd be happy to take this functionality out of mne-python and give it a new home in pybv. The order of steps could be:

1. merge this PR as an interim solution
2. move functionality from mne to pybv, still support the attribute in mne but import pybv to get it
3. deprecate the attribute in mne, and write a note in the BrainVision Raw docstr that pybv can be used to obtain impedances

Any other suggestions?

**Review by mscheltienne — APPROVED** [review]

File: mne/io/brainvision/brainvision.py (hunk 3 above)
Comment (mscheltienne) [reply mne/io/brainvision/brainvision.py]: Note that I actually stumble upon this while adding the ANT reader, and thus I added a similar `ìmpedance` attribute in the ANT reader. Maybe it should be removed as well and live in `antio`.

**Comment by larsoner** [comment]
> 1. merge this PR as an interim solution
> 2. move functionality from mne to pybv, still support the attribute in mne but import pybv to get it
> 3. deprecate the attribute in mne, and write a note in the BrainVision Raw docstr that pybv can be used to obtain impedances

I'm not sure there is a ton of value to (1) or (2) ideally (3) would make (1) and (2) obsolete. But if we don't get around to (3) for a while then (1) is useful in the interim (and done already!), so why not at least merge this.

> Note that I actually stumble upon this while adding the ANT reader, and thus I added a similar ìmpedance attribute in the ANT reader. Maybe it should be removed as well and live in antio.

Yes! In fact this makes me think that we should have in a tutorial somewhere about how you can get additional information not provided by `mne` using these other modules. It could be organized by type of info like impedance, reference schemes, etc. (my preference) or by manufacturer. Can already show `pybv` and `antio` as mentioned but also probably `mffpy`. @sappelhoff or @mscheltienne would either one of you like to get the ball rolling?

**Comment by larsoner** [comment]
Thanks @sappelhoff !

**Comment by sappelhoff** [comment]
>  Can already show pybv and antio as mentioned but also probably mffpy. @sappelhoff or @mscheltienne would either one of you like to get the ball rolling?

I am currently a bit short on time with a big backlog, so I will have to deprioritize this, but I have it on my list for pybv, at least. Would be happy if someone beat me to it though :-)

https://github.com/bids-standard/pybv/issues/122

**Pull request merged**